apps/docs/src/.vitepress/components/OnyxIconLibrary.vue

```diff
@@ -0,0 +1,47 @@
+<script lang="ts" setup>
```

Comment:
[Question] is a search / copy functionality planned for the future (out of scope of this PR) 
if so: I created https://github.com/orgs/SchwarzIT/projects/5/views/7?layout=board&pane=issue&itemId=51923734 as follow up

Reply:
Yes that would be very nice :) I did not implement it yet because we don't have a input component yet